release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -2779,6 +2779,64 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+// 4.15.49
+[id="ocp-4-15-49_{context}"]
+=== RHSA-2025:3790 - {product-title} 4.15.49 bug fix and security update
+
+Issued: 16 April 2025
+
+{product-title} release 4.15.49, which includes security updates, is now available. The list of bug fixes that are included in this update is documented in the link:https://access.redhat.com/errata/RHSA-2025:3790 [RHSA-2025:3790] advisory. The RPM packages that are included in this update are provided by the link:https://access.redhat.com/errata/RHBA-2025:3057[RHBA-2025:3792] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.15.49 --pullspecs
+----
+
+[id="ocp-4-15-49-known-issues_{context}"]
+==== Known issues
+
+* IPsec is not supported on Red Hat Enterprise Linux (RHEL) compute nodes because of a `libreswan` incompatiblility issue between a host and an `ovn-ipsec` container that exist in each compute node.  (link:https://issues.redhat.com/browse/OCPBUGS-36688 [*OCPBUGS-36688*]) 
```

Comment:
```suggestion
* IPsec is not supported on {op-system-base-full} compute nodes because of a `libreswan` incompatiblility issue between a host and an `ovn-ipsec` container that exist in each compute node.  (link:https://issues.redhat.com/browse/OCPBUGS-36688 [*OCPBUGS-36688*]) 
```